packages/agents/router/src/config.ts

```diff
@@ -166,8 +166,8 @@ export const getEnvConfig = (
 
     const maxLag = chainConfig.subgraph?.maxLag ?? MIN_SUBGRAPH_SYNC_BUFFER;
     nxtpConfig.chains[domainId].subgraph = {
-      runtime: chainConfig.subgraph?.runtime ?? chainDataForChain?.subgraph ?? [],
-      analytics: chainConfig.subgraph?.analytics ?? chainDataForChain?.analyticsSubgraph ?? [],
+      runtime: chainConfig.subgraph?.runtime ?? chainDataForChain?.subgraphs.runtime ?? [],
```

Comment:
have we updated `chainData.json` to reflect this?

Reply:
yes. we updated it in the repository. connext/chainData